servicetalk-transport-netty-internal/src/main/java/io/servicetalk/transport/netty/internal/NettyIoExecutor.java

```diff
@@ -31,6 +31,8 @@
      * Implementation of this method assumes there would be no blocking code inside the submitted {@link Runnable}s.
      * If this assumption is violated, it will impact EventLoop responsiveness and hence should be avoided.
      * @return an {@link Executor} which will use an {@link IoExecutor} thread for execution.
+     * @deprecated IoExecutor now implements {@link Executor} so this method is redundant.
      */
+    @Deprecated
```

Comment:
Consider adding a comment like `FIXME: 0.43 - remove deprecated method`, it makes easier to search what have to be cleaned in the main branch later.